src/components/title-with-description/TitleWithDescription.js

```diff
@@ -1,13 +1,17 @@
 import { Box, Typography } from '@mui/material'
 import { styles } from '~/components/title-with-description/title-with-description.styles'
 
-const TitleWithDescription = ({ title, titleStyles, description, descriptionStyles, componentStyles }) => {
+const TitleWithDescription = ({ title, titleStyles, subtitle, subtitleStyles, description, descriptionStyles, componentStyles }) => {
   return (
     <Box sx={ { ...styles.wrapper, ...componentStyles } }>
       <Typography sx={ { ...styles.title, ...titleStyles } }>
         { title }
       </Typography>
 
+      <Typography sx={ subtitle && { ...styles.subtitle, ...subtitleStyles } }>
```

Comment:
maybe you should create new component that will fit to your structure

Reply:
I remember that I have told not to reuse this component and add any subtitles to it.

Reply:
done.